microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/TracingDependencyProcessor.java

```diff
@@ -0,0 +1,38 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import org.jboss.as.server.deployment.Attachments;
+import org.jboss.as.server.deployment.DeploymentPhaseContext;
+import org.jboss.as.server.deployment.DeploymentUnit;
+import org.jboss.as.server.deployment.DeploymentUnitProcessor;
+import org.jboss.as.server.deployment.Phase;
+import org.jboss.as.server.deployment.module.ModuleDependency;
+import org.jboss.as.server.deployment.module.ModuleSpecification;
+import org.jboss.modules.Module;
+import org.jboss.modules.ModuleIdentifier;
+import org.jboss.modules.ModuleLoader;
+
+public class TracingDependencyProcessor implements DeploymentUnitProcessor {
+  public static final ModuleIdentifier EXTENSION_MODULE = ModuleIdentifier.create("org.wildfly.extension.microprofile.opentracing-smallrye");
+  public static final ModuleIdentifier MP_OT_MODULE = ModuleIdentifier.create("org.eclipse.microprofile.opentracing");
+
+  public static final Phase PHASE = Phase.DEPENDENCIES;
+  public static final int PRIORITY = 0x4000;
```

Comment:
@jmesnil and I discussed this a bit on https://github.com/wildfly/wildfly/pull/11320. We may need a follow JIRA depending on what we decide to do here.

Reply:
I changed this to be based on what was merged for MP-Config

Reply:
Ah. I missed that. It won't be allowed to have the same phase and priority. @jmesnil do you have an opinion on this? Mainly what the priority should be.